frontend/packages/console-app/src/components/network-policies/network-policy-form.tsx

```diff
@@ -264,6 +264,9 @@ export const NetworkPolicyForm: React.FC<NetworkPolicyFormProps> = ({ namespace
           header={
             <FormFieldGroupHeader
               titleText={{ text: t('public~Ingress'), id: 'ingress-header' }}
+              titleDescription={t(
+                'public~Add ingress rules to be applied to your selected pods. Traffic is allowed from pods if it matches at least one rule.',
```

Comment:
hey @OlivierCazade , one thing that changed in my last PR (merged) is that the texts for network policies are now in 'console-app' i18n namespace instead of 'public', so this should be updated (and then run again `yarn i18n` )